requirements.txt

```diff
@@ -22,12 +22,10 @@ deform==0.9.9
 elasticsearch-dsl==6.1.0
 elasticsearch==6.2.0
 enum34==1.1.6
-functools32==3.2.3.post2 ; python_version < "3.0"
 gevent==1.3.5
 greenlet==0.4.13
 gunicorn==19.9.0
 hkdf==0.0.3
-html5lib==0.999999999     # via bleach
```

Comment:
This dependency disappeared because html5lib [is now vendored into bleach](https://bleach.readthedocs.io/en/latest/changes.html#version-3-0-0-october-3rd-2018).